upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 type BootstrapChannelBuilder struct {
-	cluster *kops.Cluster
+	cluster   *kops.Cluster
+	Lifecycle *fi.Lifecycle
```

Comment:
Crazy question, should bootstrap be another lifecycle?  This would allow for the easier upgrade of stuff like CNI providers.